### ISSUE TITLE
The JS/TS language service immediately crashed 5 times. The service will not be restarted.

### ISSUE THREAD

**Issue by itsjempi** [microsoft/vscode#182468]
Type: <b>Bug</b>

showing error like this whenever starts vs code and also error and on ctrl+click  go to definition not working

VS Code version: Code 1.78.2 (b3e4e68a0bc097f0ae7907b217c1119af9e03435, 2023-05-10T14:39:26.248Z)
OS version: Windows_NT x64 10.0.22000
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz (4 x 3408)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.88GB (7.72GB free)|
|Process Argv|--crash-reporter-id 21c2eef5-baac-4f10-9bb7-58e3e56ab7cf|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
blackbox|Bla|0.10.82
htmltagwrap|bra|0.0.9
es7-react-js-snippets|dsz|4.4.3
css-flexbox-cheatsheet|dzh|3.3.2
gitlens|eam|13.6.0
vscode-html-css|ecm|1.13.1
vsc-material-theme|Equ|33.8.0
vsc-material-theme-icons|equ|2.7.5
prettier-vscode|esb|9.12.0
auto-rename-tag|for|0.1.10
vue-file-peek|how|1.0.0
jsx-format|Ica|0.0.2
vscode-gutter-preview|kis|0.30.0
color-highlight|nau|2.5.0
material-icon-theme|PKi|4.27.0
vscode-thunder-client|ran|2.6.2
LiveServer|rit|5.7.9
chatgpt|tim|1.1.2
five-server|yan|0.1.13
html-css-class-completion|Zig|1.20.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30610678
2e4cg342:30602488
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
2i9eh265:30646982
showlangstatbar:30737416
azdwalk:30721579
pythonms35:30701012
pythonfmttext:30731395
fixshowwlkth:30730052
pythongtdpathcf:30739705

```

</details>

**Comment by rwalle** [comment]
Likely related to this issue:

https://github.com/microsoft/vscode/issues/182055

You need to find out which host names need to be added to the UNC path allow list

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by itsjempi** [comment]
> Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

yaa

**Comment by mjbvz** [comment]
Please try collecting the TS Server log from the crashed instance:

1. Set `"typescript.tsserver.log": "verbose"`
1. Restart VS Code and reproduce the problem
1. Open the `TypeScript` section of the output panel
1. At the very top, find the first line that prints to path to the semantic error log file. It should look something like:

    ```
    [Info  - 19:54:59.247] <semantic> Log file: /Users/matb/Library/Application Support/Code - Insiders/logs/20200213T104930/exthost55/vscode.typescript-language-features/tsserver-log-ZT2zau/tsserver.log
    ```

    That file contains the typescript logs.

Look through that log file for the first error or stack trace you see. If you can share the log, I can also take a look to see if anything stands out

 > **⚠️Warning:** The TypeScript log may include information from your workspace, including file paths and source code. If you have any concerns about posting this publicly on Github, just let me know and we can arrange something else. On our side, we only use these logs to investigate issues like this

**Comment by kencbowman** [comment]
Experiencing the same issue: "The JS/TS language service immediately crashed 5 times. The service will not be restarted."
The issue for me was VS Code 1.78.2. Uninstalled and installed 1.77.3 and the JS/TS language service started without issue. Just to verify, I once again installed the update (i.e. 1.78.2) through VS Code (Help->Install Update...) and the issue reappeared. Again, installing version 1.77.3 fixed it.

**Comment by mjbvz** [comment]
@kencbowman Please open a new issue with the [information requested above](https://github.com/microsoft/vscode/issues/182468#issuecomment-1553436893)

**Comment by kencbowman** [comment]
@mjbvz I submitted a new issue with the requested info:

https://github.com/microsoft/vscode/issues/183276

**Comment by wengkhing** [comment]
I had also experienced this issue for some time. I've accidentally fixed the issue recently.
My repository is a monorepo setup using yarn workspaces. Recently, I did a cleanup removing all duplicated packages.
I had `typescript` installed at root `package.json` and also in all the sub projects' `package.json`, some even have different version. After removing all the `typescript` packages from sub projects and only remain the root one seems to fix the issue for me. Hopefully anyone with the same setup would find this useful.
